c/recovery.c

```diff
@@ -986,29 +1160,93 @@ int recoveryEstablishRouter(int flags) {
     ESTAEXFeedback feedback = setESTAEX(getRecoveryRouterAddress(), context,
                                         estaexFlags);
     if (feedback.returnCode != 0) {
+
+      /* Check ESTAEX RC. If it is 0x30, it means there's already an FRR set,
+       * so we must use an FRR instead of an ESTAEX.*/
+      if (feedback.returnCode == 0x30) {
```

Comment:
Could we use a constant?

Reply:
I specifically made a comment about value 0x30, and since it's localized to a single function I considered it sufficient :-) I'm happy to add a constant if that makes it more readable.

Reply:
Sorry to be pedantic! I would prefer a constant with the detailed comment you have used here but it's not a show stopper for me so I'm fine to leave it.